tests/_support/Page/Acceptance/Administrator/UserGroupPage.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class UserGroupPage extends AdminPage
+{
+    // include url of current page
+    public static $groupPageURL = 'administrator/index.php?option=com_users&view=groups';
```

Comment:
This sound redundant:

`$I->amOnPage(UserGroupPage::$groupPageURL);`

What about?

`$I->amOnPage(UserGroupPage::$url);`

that way we have a standard: every page object will have the $url property